### PR TITLE
Fixing insecure connection to ovirt

### PR DESCRIPTION
**Pull request by bkhizgiy**

### CODE REVIEW

File: ovirt_imageio/client/_ovirt.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def connect(args):
         username=args.username,
         password=args.password,
         ca_file=args.cafile,
+        insecure =not args.secure,
```
Comment (nirs) [inline ovirt_imageio/client/_ovirt.py:50]: Should be:

    insecure=not args.secure,

You can detect such errors by running tox locally.

**Review by nirs — COMMENTED** [review]
Thanks Bella!

Please add Signed-off-by to the commit message.

**Comment by aesteve-rh** [comment]
Thanks @bkhizgiy !

Unfortunately, pipelines are currently broken.
Fix is already posted: https://github.com/oVirt/ovirt-imageio/pull/177
Once fix is integrated please rebase, follow Nir's comment above, and you should be
all set :)

File: ovirt_imageio/client/_ovirt.py (hunk 1 above)
Comment (bkhizgiy) [reply ovirt_imageio/client/_ovirt.py:50]: thank you for the review, uploading now.

**Comment by bkhizgiy** [comment]
@aesteve-rh Thanks for the update!
I"ll keep track of the fix PR and rebase mine once it's in:)

File: ovirt_imageio/client/_ovirt.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ def connect(args):
         username=args.username,
         password=args.password,
         ca_file=args.cafile,
+        insecure=not args.secure,
```
Comment (nirs) [inline ovirt_imageio/client/_ovirt.py:50]: This should fix communication with engine, and communication with the host should already work.

**Review by nirs — COMMENTED** [review]
This you test this change? how? please describe how this was tested.

**Review by aesteve-rh — APPROVED** [review]

File: ovirt_imageio/client/_ovirt.py (hunk 2 above)
Comment (bkhizgiy) [reply ovirt_imageio/client/_ovirt.py:50]: Yes, I've tested the changed manually using the command `ovirt-img download-disk --engine-url=http://vm-10-122.lab.eng.tlv2.redhat.com --username=admin@internal --password-file=ovirt.pass --insecure  69f5d43f-e6c5-471c-8fe7-afac60e4e845 test` tried to initiate an Insecure transfer and it failed due to certificate issue, after I've added the above line the insecure transfer worked and the disk was downloaded successfully without any errors.

Comment (ahadas) [reply ovirt_imageio/client/_ovirt.py:50]: and even better than this - it was tested with forklift! :)

**Comment by ahadas** [comment]
> This you test this change? how? please describe how this was tested.

not only it was tested manually but also with forklift! :)

**Review by nirs — APPROVED** [review]

**Pull request merged**